chapters/hpmor-chapter-000.tex

```diff
@@ -0,0 +1,43 @@
+\chapter*{Preface by Eliezer Yudkowsky}
```

Comment:
This still seems to come out as "by Eliezer Yudkowsky"?

Reply:
sorry, forgot to push, now done

Reply:
Sorry, seems I was too quick!